lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -31,9 +31,7 @@ function PropCore.ValidSpawn(ply, model, isVehicle)
 	local limithit = playerMeta.LimitHit
 	playerMeta.LimitHit = function() end
 
-	if not PropCore.WithinPropcoreLimits() then
-		ret = false
-	elseif not (util.IsValidProp( model ) and WireLib.CanModel(ply, model)) then
+	if not (util.IsValidProp( model ) and WireLib.CanModel(ply, model)) then
 		ret = false
```

Comment:
This should be the `ret = self:throw(` right?

Reply:
That would break the detour created right before the function if it did ever error as the comment above warned

Reply:
Ah ok, what about the places ValidSpawn is used?

Reply:
Didn't check to see if it was used outside the file, only saw it once in propCanSpawn so just moved the check down. Will need to see if it is exported and then potentially add it again

Reply:
It's a custom extension so it should be isolated to it

Reply:
Checked, it is exported but not used anywhere else in wiremod. Really weird how it's exported and probably shouldn't be, but I added the check inside again just in case.